### PR TITLE
queen eye fixes

### PR DESCRIPTION
**Pull request by stanalbatross**
## Changelog

:cl:
fix: the queen eye can no longer be put in nests
fix: when emitting pheromones, if you are using the queen eye, the radial menu will appear from the eye itself
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/xenomorph/abilities/general_powers.dm [hunk 1]
```diff
@@ -335,7 +335,7 @@
 					return
 			else
 				var/static/list/phero_selections = list("Help" = image(icon = 'icons/mob/radial.dmi', icon_state = "radial_help"), "Frenzy" = image(icon = 'icons/mob/radial.dmi', icon_state = "phero_frenzy"), "Warding" = image(icon = 'icons/mob/radial.dmi', icon_state = "phero_warding"), "Recovery" = image(icon = 'icons/mob/radial.dmi', icon_state = "phero_recov"))
-				pheromone = lowertext(show_radial_menu(src, src, phero_selections))
+				pheromone = lowertext(show_radial_menu(src, src.client.eye, phero_selections))
```
Comment (Zonespace27) [inline code/modules/mob/living/carbon/xenomorph/abilities/general_powers.dm:338]:
```suggestion
				pheromone = lowertext(show_radial_menu(src, src?.client.eye, phero_selections))
```
assuming clients is always a risk

Comment (stanalbatross) [reply code/modules/mob/living/carbon/xenomorph/abilities/general_powers.dm]: yeah forgot

**Comment by Wintermote** [comment]
Does the queen eye still trigger motion detectors?

That always seemed pretty weird to me

**Comment by Diegoflores31** [comment]
i think other way would be changing all the "ismob" and make a new variable called " isliving " or smth

**Pull request merged**